concrete/concrete.go

```diff
@@ -100,34 +100,30 @@ func NewRegistry() *GenericPrecompileRegistry {
 }
 
 func (c *GenericPrecompileRegistry) index(blockNumber uint64) int {
-	for ii, startingBlock := range c.startingBlocks {
+	// Iterate over blocks and return index if found else return -1
```

Comment:
This comment is factually incorrect. Please fix it or remove it.

Reply:
agreed